examples/health-check/src/Server.hs

```diff
@@ -151,7 +154,8 @@ instance MonadMonitor m => MonadMonitor (TraceT m)
 -- Information for servant
 
 type instance AnnotatedPackage ServantRoute HealthCheckServiceFS2
-  = '[ 'AnnService "HealthCheckServiceFS2"
+  = '[ 'AnnPackage ('ServantAdditional ('Just (SwaggerSchemaUI "swagger-ui" "swagger.json")))
```

Comment:
Wow more type-level!

Reply:
Yes, that's how you add additional routes to the server which are not present in the API itself.